utils/calcRecoveryRate.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * Calculates the recovery rate and puts it in a nice looking string format.
+ * 
+ * @param  {int}     numDeaths     The number of deaths for a unit
+ * @param  {int}     numRecoveries The number of recoveries for a unit
+ * @return {String}  A parsed string percentage string of recoveries rounded to one decimal place.
+ */
+function calcRecoveryRate(numDeaths, numRecoveries) {
+    return ((numRecoveries / (numDeaths + numRecoveries)) * 100).toFixed(1) + '%';
+}
+
+/**
+ * Calculates the recovery rate from an array of data and splices the recovery rate
+ * into the middle of the array for table formatting.
+ * 
+ * @param  {Array} one data about the element we are processing. (e.g. state, country, etc.)
+ * @return {Array} A spliced array almost as it was before but now with a string formatted recovery rate.
+ */
+function spliceRecoveryRate(one) {
+    let oneRecovery = ((one[5] / (one[5] + one[3])) * 100);
```

Comment:
I assume `5` and `3` refer to rows with a certain type of data, however it's impossible to know what kind of data from just reading the code. Extracting it to const would be more informative, e.g.
`const deaths = one[5]`